packages/material/scss/_swatches.scss

```diff
@@ -0,0 +1,24 @@
+@import "palettes";
+
+
+@function get-palette($name) {
+    @return map-get($palettes, $name);
+}
+
+
+$primary-name: indigo !default;
+$secondary-name: pink !default;
+$surface-name: light !default; // light or dark
+
+
+$primary-palette: get-palette($primary-name);
+$secondary-palette: get-palette($secondary-name);
+
+
+$primary: map-get($primary-palette, 500);
+$primary-darker: map-get($primary-palette, 700);
+$primary-lighter: map-get($primary-palette, 200);
```

Comment:
These appear to be the main reason for using the named palettes, but they are not used anywhere in the theme. Perhaps these colors need to be set to other variables?

Reply:
`$primary-lighter` is use in progressbar